pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/index/reader/ForwardIndexReader.java

```diff
@@ -100,6 +100,17 @@ default int getDictIdMV(int docId, int[] dictIdBuffer, T context) {
     throw new UnsupportedOperationException();
   }
 
+  /**
+   * Reads the dictionary ids for a multi-value column at the given document id into a buffer and returns the buffer.
+   *
+   * @param docId Document id
+   * @param context Reader context
+   * @return A buffer containing the multi-value entries
+   */
```

Comment:
(minor) Suggest modifying the javadoc a little bit as the buffer is a little bit confusing. Same for other methods
```suggestion
  /**
   * Reads the dictionary ids for a multi-value column at the given document id.
   *
   * @param docId Document id
   * @param context Reader context
   * @return Dictionary ids at the given document id
   */
```

Reply:
done